src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,28 @@
-export const PostInfo = () => <>Put the post here</>;
+import { CommentList } from '../CommentList/CommentList';
+import { UserInfo } from '../UserInfo/UserInfo';
+import './PostInfo.scss';
+
+const commentSection = comments => {
+  return comments.length >= 1 ? (
+    <CommentList comments={comments} />
+  ) : (
+    <b data-cy="NoCommentsMessage">No comments yet</b>
+  );
+};
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <p>
+        {' Posted by  '} <UserInfo user={post.user} />
+      </p>
```

Comment:
in this task no such cases, but potentially post.user can be undefined. so better to render UserInfo component only if post.user exists.
```suggestion
      <p>
        {' Posted by  '} 
        { post.user && <UserInfo user={post.user} />}
      </p>
```